### ISSUE TITLE
[UX] Use proper labels in Send tab

### ISSUE THREAD

**Issue by lleaff** [aionnetwork/aion_ui#177]
The _send transaction_ form uses placeholders in place of labels.
This is generally considered bad UX, and is even more so in this particular case because:
* Some fields of the form come pre-filled so their label is never displayed.
* All fields are numbers of some sort, so the user can't guess their use from their type.
* Operations are irreversible.

![image](https://user-images.githubusercontent.com/9746807/47621342-a469b080-daf6-11e8-9c28-62f918b6ed36.png)

The _Settings_ tab uses proper labels, so it shouldn't be hard to fix.
![image](https://user-images.githubusercontent.com/9746807/47621384-48535c00-daf7-11e8-89cb-e09d379c0fc0.png)

**Comment by qoire** [comment]
@centrys-otniel We should do the following:
1) Add units to the energy row (NRG)
2) Add a drop-down for people to select the unit in which they want to enter the energy cost. Should default to our equivalent of ``gwei``.
3) Add a drop-down for people to select the unit in which they want to enter the amount of send. Should default ``Aion`` (10^18). For this we will have to resolve any conversions between units.

**Comment by centrys-otniel** [comment]
The screenshot above is a bit old. This is how the send screen looks now. The question marks also give some hints about the fields.
![screenshot from 2018-10-31 09-16-57](https://user-images.githubusercontent.com/37304045/47772041-148e5700-dcee-11e8-81da-2fe39852b0f9.png)
Also, as you can see, we already have a combo that allows selecting either the AION coin ori the configured tokens for the current account. COnsidering this, we can add something like the following:
![screenshot from 2018-10-31 09-26-21](https://user-images.githubusercontent.com/37304045/47772434-5e2b7180-dcef-11e8-808b-1ee1911ae739.png)

**Comment by lleaff** [comment]
That's a bit better, but wouldn't you agree that if there's one place where all sources of potential confusion should be removed, this is it?

I any case if labels are a definite no go then @qoire's suggestions would work too.

**Comment by centrys-otniel** [comment]
I did not mean it as "Let's not add labels". I was referring specifically to the last point of Yao's suggestions where we should add the option of sending sub-units of AION (i.e. Amp).
However, adding lables will minimise the vertical size of the text areas :grin:

**Comment by MakoMania** [comment]
As a user, I still don’t know what those 2 fields mean.   That drop down looks like a good solution but it is still a technical solution.  Any chance we can get it in layman terms for the user what it means?  For example if we have a few options for gas price, can it be indicated if it means fast medium or slow transactions?

**Comment by qoire** [comment]
@MakoMania that's noted, it just requires us to have some analysis on gas costs and their expected block inclusion times, we'll keep this suggestion in mind for future updates.

@centrys-otniel Can we also move on the labels idea suggested by @lleaff? I know in the updated builds we still don't have that.
